pybandits/cmab.py

```diff
@@ -184,7 +179,7 @@ def update(
         self.predict_actions_randomly = False
 
 
-class CmabBernoulli(BaseCmabBernoulli):
+class CmabBernoulli(BaseCmabBernoulli, TailoredBaseMab):
```

Comment:
I can understand the concern but TailoredBaseMab is still abstract (like the BaseMab), thus not implementing any of the abstract methods. Consequently, there are no conflicts between the "parents" of CmabBernoulli. To this end, I don't see how to separate their duties any further. If you have something in mind, do let me know 🙏

Reply:
Why do you need to have 2 children of a base class (so 2 diverging behavior from a common point) that then need to be merged back? 

You should be able to organize the code in a way to have a single line of inheritance and whatever utility method coming from a separate entity, or to have N unrelated lines of inheritance finally merged in the implementation class with multiple inheritance

Reply:
Now I get you, thanks for this one!
I think of the following solution:
Get rid of the TailoredBaseMab class. Move cold_start to the BaseMab. It would then generate the strategy and inject it as a parameter of the __init__ to produce the instance. WDYT?

Reply:
Now I fixed it. I see how much the TailoredBaseMab was redundant. Thanks!

Reply:
Very good approach 😄 happy to help